docs/csharp/whats-new/csharp-12.md

```diff
@@ -1,17 +1,18 @@
 ---
 title: What's new in C# 12 - C# Guide
 description: Get an overview of the new features in C# 12.
-ms.date: 04/04/2023
+ms.date: 09/23/2023
 ---
 # What's new in C# 12
 
 Some C# 12 features have been introduced in previews. You can try these features using the latest [Visual Studio preview](https://visualstudio.microsoft.com/vs/preview/) or the latest [.NET 8 preview SDK](https://dotnet.microsoft.com/download/dotnet).
 
-- [Primary constructors](#primary-constructors) - Introduced in Visual Studio 17.6 preview 2.
-- [Optional parameters in lambda expressions](#default-lambda-parameters) - Introduced in Visual Studio 17.5 preview 2.
-- [Alias any type](#alias-any-type) - Introduced in Visual Studio 17.6 preview 3.
-- [Inline arrays](#inline-arrays) - Introduced in Visual Studio 17.7 preview 3.
+- [Primary constructors](#primary-constructors) - Introduced in Visual Studio 17.6, preview 2.
```

Comment:
Technically these should be patterned like this (note casing):

Visual Studio version 17.6 Preview 3